packages/@aws-cdk/aws-iot/README.md

```diff
@@ -83,3 +83,16 @@ new iot.TopicRule(this, 'TopicRule', {
 ```
 
 See also [@aws-cdk/aws-iot-actions](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-iot-actions-readme.html) for other actions.
+
+## Thing
+
+```ts
+new iot.Thing(this, 'Thing', {
+  thingName: 'my-thing', // optional property, default is a generated name
+  attributes: { // optional property, default is none
```

Comment:
why/when to use attributes?

Reply:
Usually `attributes` store about static information about the thing, such as product name, serial number, lot number or manufacturer. And it is used for search things on web console and cli.